charts/devtron/values.yaml

```diff
@@ -369,29 +406,40 @@ notifier:
 minio:
   enabled: false
   replicaCount: 1
-  image: "quay.io/devtron/minio:RELEASE.2021-02-14T04-01-33Z"
+  image: "minio:RELEASE.2021-02-14T04-01-33Z"
   imagePullPolicy: IfNotPresent
-  mbImage: "quay.io/devtron/minio-mc:RELEASE.2021-02-14T04-28-06Z"
+  mbImage: "minio-mc:RELEASE.2021-02-14T04-28-06Z"
   mbImagePullPolicy: IfNotPresent
+  gatewayImage: "minio:RELEASE.2020-12-03T05-49-24Z"
   persistence:
     storage: "50Gi"
 # Change below values for workflow controller
 workflowController:
+  registry: "quay.io/argoproj"
   # Set this to true if you have IMDSv2 enforced or IMDSv1 and v2 on your AWS EKS cluster and false if you are using IMDSv1 with token hop limit set to 1
   IMDSv2Enforced: true
-  image: "quay.io/argoproj/workflow-controller:v3.4.3"
-  executorImage: "quay.io/argoproj/argoexec:v3.4.3"
+  image: "workflow-controller:v3.4.3"
+  executorImage: "argoexec:v3.4.3"
+  IMDSv1: "workflow-controller:v3.0.7"
+  IMDSv1executorImage: "argoexec:v3.0.7"
```

Comment:
change it to `IMDSv1ExecutorImage` in both bom and values as well as template

Reply:
Done